jenkins/e2e-image/Dockerfile

```diff
@@ -52,4 +52,7 @@ ADD ["e2e-runner.sh", \
     "/workspace/"]
 RUN ["chmod", "+x", "/workspace/get-kube.sh", "/workspace/upload-to-gcs.sh", "/workspace/sh2ju.sh"]
 
+ADD ["configure-image.sh", "/tmp"]
+RUN ["/tmp/configure-image.sh"]
```

Comment:
also, I wonder if this should instead be a separate Docker image which is based on this one, rather than adding more heft to `kubekins-e2e`.